drawee/src/main/java/com/facebook/drawee/drawable/ScalingUtils.java

```diff
@@ -83,11 +82,18 @@
      * It is guaranteed that the focus point will be visible and centered as much as possible.
      * If the focus point is set to (0.5f, 0.5f), result will be equivalent to CENTER_CROP.
      */
-    FOCUS_CROP
+    FOCUS_CROP,
+
+    /**
+     * keep height dimension fix exactly and aspect ratio is preserved.
```

Comment:
```
/**
 * Scales the image so that the y dimension fits exactly and aspect ratio is preserved.
 * Image is horizontally centered in the parent.
 */
```